modules/cli/src/main/scala/scala/cli/commands/setupide/SetupIdeOptions.scala

```diff
@@ -2,11 +2,28 @@ package scala.cli.commands.setupide
 
 import caseapp.*
 
-import scala.cli.commands.shared.{HasSharedOptions, SharedBspFileOptions, SharedOptions}
+import scala.cli.ScalaCli.{baseRunnerName, fullRunnerName}
+import scala.cli.commands.shared.{
+  HasSharedOptions,
+  HelpMessages,
+  SharedBspFileOptions,
+  SharedOptions
+}
 import scala.cli.commands.tags
 
 // format: off
-@HelpMessage("Generate a BSP file that you can import into your IDE")
+@HelpMessage(
+  s"""Generates a BSP file that you can import into your IDE.
+     |
+     |The `setup-ide` sub-command allows to pre-configure a $fullRunnerName project to import to an IDE with BSP support.
+     |It is also ran implicitly when `compile`, `run`, `shebang` or `test` sub-commands are called.
+     |
+     |The pre-configuration should be saved in a BSP json connection file under the path:
+     |    {project-root}/.bsp/$baseRunnerName.json
```

Comment:
From the user's point of view, this information is irrelevant. I would suggest to  remove it

Reply:
We do mention it in our [IDE support docs](https://scala-cli.virtuslab.org/docs/guides/ide)... why not mention it here?
by verifying if the `.bsp` directory has been created, it's easier for the user to check if `setup-ide` has been run in a directory. it also marks the project root to be opened in his IDE.
IMO won't hurt to mention it in the help as well...